documentation/modules/virt-migration-workflow.adoc

```diff
@@ -30,10 +32,30 @@ The workflow describes the following steps:
 +
 *After the VM disks are transferred:*
 
-. The `Migration Controller` service creates a `conversion` pod with the PVCs attached to it.
+. The `Migration Controller` service creates a `conversion` pod with the PVCs attached to it when importing from VMWare.
 +
 The `conversion` pod runs `virt-v2v`, which installs and configures device drivers on the PVCs of the target VM.
++
+. The `Migration Controller` service creates a `VirtualMachine` CR for each source virtual machine (VM), connected to the PVCs.
+
+. When the target VM is powered on, the `KubeVirt Controller` service creates a `virt-launcher` pod and a `VirtualMachineInstance` CR.
```

Comment:
we start the vm automatically on the destination when the users asks us to so for this particular case I'd keep it, otherwise what happens after the migration is finished is out of scope. so instead of "When the target VM is powered on" I'd write (but need to double check me on this): "when the option to restore the power state of the VM is checked and the VM was running on the source environment, the `Migration Contoller` powers on the VM .."

Reply:
the user can't ask for it. if the source vm is powered on, it will automatically power-on on destination

Reply:
indeed, I had a vague memory of being able to choose - not sure why..